### ISSUE TITLE
Permission denied when trying to create database

### ISSUE THREAD

**Issue by haarp** [n76/Local-GSM-Backend#62]
I am testing this backend, but I am unable to create a database. Attempts at downloading it immediately result in: open failed: EACCES (Permission denied)

from logcat:

    W/System.err( 2163): java.io.FileNotFoundException: /storage/emulated/0/Android/data/org.fitchfamily.android.gsmlocation/files/lacells_gen.log: open failed: EACCES (Permission denied)

Which is weird. I have no /storage/emulated/0. My internal "sd card" is 
/storage/emulated/legacy.

This is on Cyanogenmod 12.1

**Comment by JERW86** [comment]
@n76 and @haarp 

I also have this issue on CM 12.1 after updating to 1.3.6.

My internal "sd card" is /storage/emulated/0. 
Phone: SGH-i747

Logcat: [GSM Location Service 1.3.6 error.txt](https://github.com/n76/Local-GSM-Backend/files/190110/GSM.Location.Service.1.3.6.error.txt) and screenshot:
![gsm location service 1 3 6 error](https://cloud.githubusercontent.com/assets/16406776/14054139/31201a26-f2ad-11e5-8b76-8f24aa047553.png)

Also my database, located in /storage/emulated/0/Android/data/org.fitchfamily.android.gsmlocation/files/lacells_gen.log, that I generated using 1.3.4 earlier this month, is no longer recognized by the app. When I use my file browser, the files are still in the above location, but the app says "No Database Found. Create Database."

I have reverted back to 1.3.4 for now.
